hazelcast/src/main/java/com/hazelcast/query/impl/getters/MultiResult.java

```diff
@@ -70,6 +70,8 @@
 
     private List<T> results;
 
+    private boolean nullOrEmptyTarget;
```

Comment:
hmm weird, I didn't delete it on purpose, somehow it disappeared... Sure I will add it.